python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -592,34 +592,25 @@ def forward(self, x: torch.Tensor):
 
 
 class BF16Linear(nn.Linear):
-    def __init__(self, input_features, output_features, bias=True,
+    def __init__(self, input_features, output_features, qtype, bias=True,
```

Comment:
no need for `qtype` here?

Reply:
Have remove all unnecessary qtype passing, only keep a default qtype of `BF16Linear`.